### PR TITLE
make license check from mm-plugin-api

### PR DESCRIPTION
**Pull request by cjmartian**
#### Summary

Adds license checks from mattermost-plugin-api 

#### Ticket Link

https://github.com/mattermost/mattermost-plugin-msteams-meetings/issues/17

### CODE REVIEW

**Comment by cjmartian** [comment]
@hanzei I have what I think is an appropriate PR. It looks like the CI is failing at the `Check git diff` step. I modified `go.mod` and `go.sum` which are the two files it's diffing. I'm not sure if modifying those files is forbidden. Otherwise, it looks like it's passing the rest of the tests.

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,8 @@ module github.com/mattermost/mattermost-plugin-msteams-meetings
 go 1.12
 
 require (
-	github.com/mattermost/mattermost-server/v5 v5.27.0
+	github.com/mattermost/mattermost-plugin-api v0.0.12-0.20200908141642-fbaca40730c4
```
Comment (hanzei) [inline go.mod:6]: Nit pick: I think you can use the latest release `v0.0.12` directly here

**Review by hanzei — CHANGES_REQUESTED** [review]
The changes look good :+1: 

For CI to pass you need to run `go mod tidy` and commit the changes.

**Review by larkox — APPROVED** [review]
LGTM. Thanks! :tada:

File: go.mod (hunk 1 above)
Comment (cjmartian) [reply go.mod:6]: I'll make that change, thank you for pointing that out

Comment (cjmartian) [reply go.mod]: Could we do the same with https://github.com/mattermost/mattermost-plugin-msteams-meetings/pull/18/files#diff-37aff102a57d3d7b797f152915a6dc16R7

**Review by hanzei — APPROVED** [review]

**Comment by hanzei** [comment]
Thanks for the contribution @cjmartian :+1:

**Comment by cjmartian** [comment]
Anytime, thank you for giving me a review and helping me get this one in! @hanzei

**Pull request merged**